dockerfiles/Dockerfile.router

```diff
@@ -1,39 +1,43 @@
 ARG DEBUG_IMAGE
 # Build is required to extract the release files
-FROM --platform=linux/amd64 alpine:latest AS build
+FROM --platform=linux/amd64 debian:bullseye-slim AS build
 
-ARG ROUTER_RELEASE
+ARG ROUTER_RELEASE=latest
```

Comment:
I think this default makes a lot of sense and offers a nice user experience and pairs nicely with the capability of our `router` install endpoint.